src/components/About.tsx

```diff
@@ -10,8 +10,8 @@ export default function About() {
         <div className="py-12 md:py-6">
           <div className="max-w-3xl mx-auto text-center pb-12 md:pb-20">
             <img src={logo} alt="Alexandria logo" className="mx-auto h-40 w-auto pb-8"/>
-            <p className="text-xl text-gray-600">Research shows language acquisition occurs when we comprehend messages alongside its surrounding context. </p>
-            <p className="mt-10 text-xl text-gray-600">This is where Alexandria comes in.</p>
+            <p className="text-xl text-gray-600">Languages can be learnt in many different ways. Of of them is by reading texts, translating them word by word at first, and over time becoming more and more familiar with the vocubulary.</p>
+            <p className="mt-10 text-xl text-gray-600">Alexandria was build to make this process easier for you.</p>
```

Comment:
This doesn't really work for me. It lacks tempo and rhythm of a lead-in piece of text.

Reply:
Ok realize that feedback was not particularly helpful. Let me see if I can capture the intensions of the sentence based on my own seemingly rando criteria.